src/components/link/link.js

```diff
@@ -112,78 +112,109 @@ export const omitLinkProps = propsToOmit => {
   }, {})
 }
 
-const clickHandlerFactory = ({ disabled, tag, href, suppliedHandler, parent }) => {
```

Comment:
@tmorehouse I think we can remove the `pickLinkProps()` and `omitLinkProps()` utils and their tests as they aren't used anywhere in the codebase.

Reply:
Possibly... but I though they were used somewhere....

Reply:
I thought so too in the first place but I looked it up and haven't found any reference.

Reply:
They may have been used previously... and then removed.

And we have the newer `omit` helper in `utils/object.js` as well.

Reply:
Yeah. Will save some bytes too...

Reply:
Removed the two helper utils... lets see if anything complains.